drizzle.config.ts

```
@@ -0,0 +1,67 @@
+//         dbCredentials: {
+//           url: getLocalD1DB(),
+//         },
+//       }),
+// });
+
+import { config } from "dotenv";
+import { defineConfig } from "drizzle-kit";
+
+config({ path: ".dev.vars" });
+
+export default process.env.LOCAL_DB_PATH
```

ESLint

[error] Unexpected use of process.env.

(node/no-process-env)